src/screens/Departures/AllStopsOverview.tsx

```diff
@@ -1,102 +1,62 @@
+import {Location} from '@atb/favorites/types';
+import ScreenReaderAnnouncement from '@atb/components/screen-reader-announcement';
+import StopPlaces from '@atb/screens/Departures/components/StopPlaces';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {View} from 'react-native';
+import {LocationInput, Section} from '@atb/components/sections';
+import {Location as LocationIcon} from '@atb/assets/svg/mono-icons/places';
+import FavoriteChips from '@atb/favorite-chips';
 import {NearestStopPlacesQuery} from '@atb/api/types/generated/NearestStopPlacesQuery';
-import DeparturesTexts from '@atb/translations/screens/Departures';
-import {DeparturesStackParams} from '.';
-import StopPlaceItem from './components/StopPlaceItem';
-import {useServiceDisruptionSheet} from '@atb/service-disruptions';
-import {useDoOnceWhen} from '@atb/screens/utils';
-
-const DateOptions = ['now', 'departure'] as const;
-type DateOptionType = typeof DateOptions[number];
-export type SearchTime = {
-  option: DateOptionType;
-  date: string;
-};
+import {DeparturesStackParams} from '@atb/screens/Departures/index';
+import {StackNavigationProp} from '@react-navigation/stack';
+import {RootStackParamList} from '@atb/navigation';
 
-type DeparturesRouteName = 'DeparturesRoot';
-const NearbyRouteNameStatic: DeparturesRouteName = 'DeparturesRoot';
+type DeparturesRouteProps = RouteProp<
+  DeparturesStackParams,
+  'AllStopsOverview'
+>;
 
-export type DepartureScreenNavigationProp = CompositeNavigationProp<
+type DepartureScreenNavigationProp = CompositeNavigationProp<
   StackNavigationProp<DeparturesStackParams>,
   StackNavigationProp<RootStackParamList>
 >;
 
-export type NearbyPlacesParams = {
+export type AllStopsOverviewParams = {
   location: Location;
 };
 
-export type DeparturesProps = RouteProp<
-  DeparturesStackParams,
-  DeparturesRouteName
->;
-
 type RootProps = {
   navigation: DepartureScreenNavigationProp;
-  route: DeparturesProps;
+  route: DeparturesRouteProps;
 };
 
-export default function NearbyPlacesScreen({navigation}: RootProps) {
+export const AllStopsOverview = ({navigation}: RootProps) => {
   const {status, location, locationEnabled, requestPermission} =
     useGeolocationState();
 
-  if (!status) {
-    return <Loading />;
-  }
-
-  return (
-    <PlacesOverview
-      requestGeoPermission={requestPermission}
-      hasLocationPermission={locationEnabled && status === 'granted'}
-      currentLocation={location || undefined}
-      navigation={navigation}
-    />
-  );
-}
-
-type PlacesOverviewProps = {
-  currentLocation?: GeoLocation;
-  hasLocationPermission: boolean;
-  requestGeoPermission: RequestPermissionFn;
-  navigation: DepartureScreenNavigationProp;
-};
-
-const PlacesOverview: React.FC<PlacesOverviewProps> = ({
-  requestGeoPermission,
-  currentLocation,
-  hasLocationPermission,
-  navigation,
-}) => {
+  const requestGeoPermission = requestPermission;
+  const currentLocation = location || undefined;
```

Comment:
Are these needed or could we just either rename in destructuring or just use the names from `useGeolocationState`?